Pipfile

```diff
@@ -20,24 +19,22 @@ flake8-comprehensions = "*"
 flake8-debugger = "*"
 flake8-deprecated = "*"
 flake8-docstrings = "*"
-flake8-isort = "*"
```

Comment:
We have a dedicated pre-commit command for `isort` so no need to run it (again) with flake8.